lumicks/pylake/kymo.py

```diff
@@ -133,6 +133,18 @@ def _timestamps(self, sample_timestamps):
         return reconstruct_image(sample_timestamps, self.infowave.data,
                                  self.pixels_per_line, reduce=np.mean).T
 
+    @property
+    def pixelsize_um(self):
+        return [axes["pixel size (nm)"] / 1000 for axes in self._ordered_axes()]
```

Comment:
Needs a docstring to indicate that this is an array equal to the number of axes.

Reply:
Added!